DigitalOcean.API/Models/Responses/Size.cs

```diff
@@ -17,12 +18,12 @@ public class Size {
         /// This attribute describes the monthly cost of this Droplet size if the Droplet is kept for an entire month. The value is
         /// measured in US dollars.
         /// </summary>
-        public float PriceMonthly { get; set; }
+        public Decimal PriceMonthly { get; set; }
 
         /// <summary>
         /// This describes the price of the Droplet size as measured hourly. The value is measured in US dollars.
         /// </summary>
-        public float PriceHourly { get; set; }
+        public Decimal PriceHourly { get; set; }
```

Comment:
These properties represent currencies, Decimal type is more appropriate than float.